src/System.Private.CoreLib/src/System/Threading/RuntimeThread.cs

```diff
@@ -2,33 +2,123 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
+using System.Diagnostics;
+using System.Runtime;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.Threading
```

Comment:
Made the change. Chatted with @AntonLapounov offline, removed RuntimeThread from System.Threading to avoid name conflicts that would require renaming all uses inside System.Threading temporarily.